proto/sentry_protos/snuba/v1/endpoint_time_series.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "google/protobuf/timestamp.proto";
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+message TimeSeriesRequest {
+  RequestMeta meta = 1;
+  TraceItemFilter filter = 2;
+  repeated AttributeAggregation aggregations = 3;
+  uint64 granularity_secs = 4;
+  repeated AttributeKey group_by = 5;
+}
+
+message TimeSeries {
+  string label = 1;
+  // the names and values of the attribute keys which
+  // were in the group by
+  map<string, string> group_by_attributes = 2;
+  // time bucket for each data point in the timeseries
+  repeated google.protobuf.Timestamp buckets = 3;
```

Comment:
Are you enforcing somewhere that every timeseries has a data point for every bucket? Otherwise you can get sparse time series that don't have data for specific buckets.